pdns/ixfrdist.cc

```diff
@@ -806,14 +922,24 @@ static void handleUDPRequest(int fd, boost::any&) {
     return;
   }
 
-  if (!allowedByACL(saddr)) {
+  MOADNSParser mdp(true, string(&buf[0], static_cast<size_t>(res)));
```

Comment:
I'm not thrilled by the parsing happening before checking the ACL, but I can live with it.

Reply:
hmm, I actually have two NMGs that, together, cover all possible legitimate sources, I'll improve this

Reply:
in fact the TCP path already does that!